tests/sample/droppacket.c

```diff
@@ -33,14 +25,6 @@ DropPacket(xdp_md_t* ctx)
     ETHERNET_HEADER* ethernet_header = NULL;
     long key = 0;
 
-    uint32_t* interface_index = bpf_map_lookup_elem(&interface_index_map, &key);
```

Comment:
This was for testing if xdp_ctx is passing if_index. I think we should keep this - at least for the case where program is attached with ifindex = 0.

Reply:
When Alan and I spoke we said we should delete it.  It's just a sample after all, so it shouldn't be that important.   Removal simplifies the sample so I'd prefer to keep it.  If you have a technical reason it's actually important, we can discuss and possibly change it back.

Reply:
The `xdp_md_ctx` parameter has `ifIndex` as a field - and there can be xdp programs that can inspect the ifindex and take decision based on it. So I think we should keep an active test case where the xdp program checks the ifindex. I think this was the only sample xdp program that inspected the ifindex field of the xdp_md_ctx param.

Reply:
Good point.